### PR TITLE
Allow users to sort selection

### LINKED ISSUE #24: order of elements
I think it would be useful to allow user to change the order of elements on the list - sometimes order matter.

### PR DESCRIPTION
**Pull request by j6s**
This adds a second column of buttons to the right of the selected items list that allows sorting the currently active item up or down in that list while also reordering the underlying `select` at the same time.

The feature is turned off by default and can be enabled by passing `sortable: true`.

Resolves #24

### CODE REVIEW

**Comment by mxmissile** [comment]
Why would this ever not get merged?

**Comment by j6s** [comment]
For what it's worth: I have been using this in production for about a year without issues at this point

File: src/dual-listbox.js [hunk 1]
```diff
@@ -428,6 +438,117 @@ class DualListbox {
...
+     *
+     * @private
+     * @param {string} direction
+     * @return {int[]}
+     */
+    _findSelected(direction) {
+        const oldIndex = this.selected.findIndex(element => element.classList.contains('dual-listbox__item--selected'));
+
+        let newIndex = oldIndex;
+        if ('up' === direction && oldIndex > 0) {
```
Comment (j6s) [inline src/dual-listbox.js:495]: This should probably use the constants declared above

**Comment by JostCrow** [comment]
Sorry for the delay. I am looking into it now.

**Review by JostCrow — APPROVED** [review]

**Pull request merged**